libs/protocol-client/src/lib/client/user.ts

```diff
@@ -38,8 +39,11 @@ export class User extends BaseClient {
 }
 
 class GuildUser extends BaseClient {
+  public async create(args: CreateGuildUserMutationVariables) {
+    return await this.sdk.createGuildUser(args);
```

Comment:
Don't we need to add this to the graphql-shield? And also we probably want to have some permissioning on this so that only a user can add themselves to a guild. We also need to prevent the user from adding a membership status. We should take the default status.

Reply:
Just noticed that it merged to prod. Thanks for reminding me!